scripts/falcoctl/falcoctl.yaml.in

```diff
@@ -1,5 +1,5 @@
 driver:
-    type: "kmod"
+    type: [@FALCOCTL_DRIVER_TYPES@]
```

Comment:
The list of allowed types is dynamically set, eg: if modern bpf is not built, it won't be present.

Reply:
:+1: 

Order is correct :point_down: 
https://github.com/falcosecurity/falco/pull/3154/files#diff-1fa114fd37754ac397e510000838b3f7bc8cbe1c517d67e680dd4644cfc99c21R51-R60